runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldSwitcherPanel.java

```diff
@@ -159,24 +159,24 @@ void updateList()
 			switch (orderIndex)
 			{
 				case PING:
-					return Integer.compare(r1.getPing(), r2.getPing()) * (ascendingOrder ? 1 : -1);
+					// Leave worlds with unknown ping at the bottom
+					return r1.getPing() > 0 == r2.getPing() > 0

```

Comment:
This should be abstracted out into logic if statements IMO instead of nested ternaries.

Reply:
runelite-client can use Guava, so https://google.github.io/guava/releases/snapshot-jre/api/docs/com/google/common/collect/ComparisonChain.html would make this super clear what's going on.